src/table/context/useContextSelector.ts

```diff
@@ -38,7 +31,7 @@ export function useContextSelector<T, TSelected>(context: Context<T>, selector:
     return () => {
       listeners.delete(listener);
     };
-  }, [listeners]);
+  }, [latestSelectedState, latestSelector, listeners]);
```

Comment:
I think since `latestSelectedState` and `latestSelector` becomes a ref state now (by using `useMutableProp()`), does passing it as dependency in this effect makes any difference? IMO it's extra in this deps list

Reply:
It does not but eslint doesn't understand that they are refs. So instead of disable the eslint rule here. I just added them as a deps.

It would work, as in no need to have them in deps array, if I declared the ref inside this hook. But since I do it via `useMutableProp` it does not work anymore